.eslintrc.yml

```diff
@@ -10,6 +11,8 @@ env:
   node: true
 
 rules:
+  indent: [2, "tab"]
```

Comment:
Does this rule change specify tabs instead of spaces? I'd rather the linting wasn't changed